Update DOCS.md (#3221)
home-assistant/addons: ssh/DOCS.md

You can access the web terminal by clicking the "Open Web UI" button on this add-on's Info tab. If you set the "Show in sidebar" setting (found on the same Info tab) to "on", a shortcut is added to the sidebar allowing you to access the web terminal quickly.

To copy text from the Web UI:
- hold down the SHIFT key

> - select the text you want to copy
> - when you release the left mouse button the text will copy to your system clipboard

Reviewer: ```suggestion
- select the text you want to copy using your mouse
- releasing the left mouse button will copy the text to your system clipboard
```

> 

Reviewer: ```suggestion
- select the text you want to copy using your mouse
- releasing the left mouse button will copy the text to your system clipboard
```

> To copy text from the Web UI:
> - hold down the SHIFT key
> - select the text you want to copy
> - when you release the left mouse button the text will copy to your system clipboard
> 
> To paste text in the Web UI press SHIFT + INSERT

Reviewer 2: ```suggestion
You can copy text by holding down the shift key and then selecting the text to copy. Releasing the mouse button will copy the text to your system clipboard.
Use SHIFT + INSERT to paste text to the terminal.
```
Rewording suggestion.

Reviewer: I think the step by step style is kinda nice. But I like some your suggestion style wise. Made a separate suggestion.

> - hold down the SHIFT key
> - select the text you want to copy using your mouse
> - releasing the left mouse button will copy the text to your system clipboard
> 
> To paste text in the Web UI press SHIFT + INSERT

Reviewer 3: ```suggestion
1. Hold down the SHIFT key.
2. Select the text you want to copy using your mouse.
3. To copy the text to your system clipboard, release the left mouse button.
4. To paste text in the Web UI, press SHIFT + INSERT.
```

Reviewer: I think copy and pasting text are (potentially) separate things you want to do in the shell, so I would not lump those together in one sequence.

Reviewer 3: right

Reviewer: One could argue that "select the text" already contains the left mouse release... But this is maybe a bit detail. I kinda liked the passive mood though for that reason, just explaining when the copying exactly happens. But it gets into details here :sweat_smile:
